0.6/docker-entrypoint.sh

```diff
@@ -40,8 +40,13 @@ fi
 # CONSUL_CONFIG_DIR isn't exposed as a volume but you can compose additional
 # config files in there if you use this image as a base, or use CONSUL_LOCAL_CONFIG
 # below.
-CONSUL_DATA_DIR=/consul/data
-CONSUL_CONFIG_DIR=/consul/config
+if [ -z "$CONSUL_DATA_DIR" ]; then
```

Comment:
I pretty much agree, but I prefer to take pity on the misguided. e.g. if they're trying to automate a `docker run` in bash and forget the `nounset`.